pkg/importer/dataStream.go

```diff
@@ -896,7 +900,7 @@ func (d *DataStream) convertQcow2ToRaw(src, dest string) error {
 
 // Copy endpoint to dest based on passed-in reader.
 func (d *DataStream) copy(dest string) error {
-	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
+	if d.VolumeMode != string(v1.PersistentVolumeBlock) && util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
```

Comment:
This should not be needed, having a block device should not matter on if we clean up the scratch space.

Reply:
Why do you want to clean up the scratch space if this code (Phase I) does not know what is scratch space. There is no scratch space to clean up...

Reply:
We should clean up scratch space if it is used, regardless of if we are using a block device or not. So if scratch space is available 
```
util.GetAvailableSpace(d.ScratchDataDir) > int64(0) 
```
Then we clean it up, block or no block.

Reply:
This feature does not use scratch space. I do not know what is scratch space at that point.